velox/exec/Task.h

```diff
@@ -646,9 +660,14 @@ class Task : public std::enable_shared_from_this<Task> {
       core::PlanFragment planFragment,
       int destination,
       std::shared_ptr<core::QueryCtx> queryCtx,
+      ExecutionMode mode,
```

Comment:
Do we need backward compatibility? Otherwise it will break Prestissimo which uses the old Task::create API? Thanks!